view/theme/frio/css/image_grid.css

```diff
@@ -0,0 +1,25 @@
+* {
```

Comment:
Have you tested this on the rest of the web UI to check for potential regression?

Reply:
I am not sure what exactly you mean by "the rest of the web UI". In any way, I tested Frio and Vier successfully.

Reply:
I realize now that this stylesheet would have only been included in the Network page, but after you move it to `view/global.css`, it will be active on all pages which might modify their display.